src/noexception.cc

```diff
@@ -0,0 +1,87 @@
+/* 
+ * Copyright 2010-2011 PathScale, Inc. All rights reserved.
```

Comment:
I don't think PathScale owns this (not least because PathScale Inc went bankrupt a couple of years ago).

Reply:
Thanks, fixed this.